storage-bigtable/src/lib.rs

```diff
@@ -141,6 +141,7 @@ impl From<ConfirmedBlock> for StoredConfirmedBlock {
             parent_slot,
             transactions,
             rewards,
+            num_partitions: _num_partitions,
```

Comment:
do you mind updating the comment above on `StoredConfirmedBlock` in another PR to say that it should not store any new fields since we don't store blocks this way anymore?